specification/ingest/_types/Processors.ts

```diff
@@ -528,13 +547,66 @@ export class CircleProcessor extends ProcessorBase {
   target_field?: Field
 }
 
+export class CommunityIDProcessor extends ProcessorBase {
+  /**
+   * Field containing the source IP address.
+   */
+  source_ip?: string
```

Comment:
Applies to all optional fields:

Please add a `@server_default` tag for the ones that have documented default values. In this case it's `source.ip` for example.

Reply:
Added as based on the documentation.